### PR TITLE
Change categories of some weapons

### PR DESCRIPTION
**Pull request by Koud-Wind**
Note: https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/161 needs to be completed first

### CODE REVIEW

File: src/main/java/com/paneedah/mwc/items/guns/SupernovaFactory.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public Item createGun(CommonProxy commonProxy) {
         .withCrosshairRunning("Running")    
         .withShellCasingEjectEnabled(false)
         .withCrosshairZoomed("Sight")
-        .withInaccuracy(15)
+        .withInaccuracy(13)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/SupernovaFactory.java:48]: please revert this change (I will do extended research on balance of weapons at a later date)

File: src/main/java/com/paneedah/mwc/items/guns/Remington870Factory.java [hunk 2]
```diff
@@ -21,8 +21,8 @@ public Item createGun(CommonProxy commonProxy) {
 
         .withName("remington870")
 //      .withAmmo(CommonProxy.Remington870Mag)
-        .withAmmoCapacity(4)
-        .withMaxBulletsPerReload(4)
+        .withAmmoCapacity(5)
+        .withMaxBulletsPerReload(5)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/Remington870Factory.java:25]: This was correct, so please revert this change.
Refer to https://www.remarms.com/shotguns/pump-action/model-870/

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java [hunk 3]
```diff
@@ -1044,7 +1044,7 @@ else if(model instanceof AcogReticle) {
                   }, 250, 0))
                   
           .build())
-		.withSpawnEntityDamage(7f)
+		.withSpawnEntityDamage(10f)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:1047]: please revert this change (I will do extended research on balance of weapons at a later date)

File: src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java [hunk 4]
```diff
@@ -71,7 +71,7 @@ public Item createGun(CommonProxy commonProxy) {
         ))
 		.withInformationProvider(stack -> Arrays.asList(
 		"Type: .50 Beowulf Rifle", 
-		"Damage: 7", 
+		"Damage: 10",
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java:74]: please revert this change (I will do extended research on balance of weapons at a later date)

File: src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java [hunk 5]
```diff
@@ -1008,7 +1008,7 @@ else if(model instanceof AcogReticle) {
                                 }, 250, 0))
 
                         .build())
-                .withSpawnEntityDamage(5.4f)
+                .withSpawnEntityDamage(6.4f)
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java:1011]: please revert this change (I will do extended research on balance of weapons at a later date)

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Comment by Koud-Wind** [comment]
I think authenticity should be a priority because we have a balance pack, and if players feel that balance is not good, they can change the balance themselves. It's better to reduce the head burst damage by a little bit
I don't want VSS to become PUBG like that

**Review by Desoroxxx — APPROVED** [review]

**Pull request merged**